src/scheduling/schedule_config.py

```diff
@@ -80,6 +118,18 @@ def as_serializable_dict(self):
             'start_datetime': date_utils.to_iso_string(self.start_datetime)
         }
 
+        if self.end_option == 'end_datetime':
+            result['end_option'] = self.end_option
+            result['end_arg'] = date_utils.to_iso_string(self.end_arg)
+        elif self.end_option == 'max_executions':
+            result['end_option'] = self.end_option
+            result['end_arg'] = self.end_arg
+        elif self.end_option == 'never':
+            result['end_option'] = 'never'
```

Comment:
Should we just skip it? As in a comment above, I would suggest to completely omit `end_...` configuration if it's not needed.